<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPanelEditorDialog</class>
 <widget class="QMainWindow" name="AddPanelEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edX">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edY">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnSetName">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Set Panel Name</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddText">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edText"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Select the type of icon to add to panel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbType">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string extracomment="RHt">Right Hand Turnout</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Left hand Turnout</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Slip/3-Way Turnout/Scissors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sensor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Signal Head</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Signal Mast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Memory</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reporter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Background</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Multisensor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RPS Reporter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fast Clock</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Icon</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="chkPopup">
      <property name="text">
       <string>Panel items popup menus active</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkReposition">
      <property name="text">
       <string>All panel items can be repositioned</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkControlLayout">
      <property name="text">
       <string>Panel items control layout</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkShowHidden">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Show all hidden items </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkShowTooltips">
      <property name="text">
       <string>Show tooltips for all items</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkHasMenu">
      <property name="text">
       <string>Panel has menu</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Panel scrollbars</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbScrollBars">
        <item>
         <property name="text">
          <string>No Scrollbars</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Both scrollbars</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horizontal only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vertical only</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
